doc/api/assert.md

```diff
@@ -147,6 +147,67 @@ try {
 }
 ```
 
+## Class : assert.Tracker
+
+### `new assert.Tracker`
+<!-- YAML
+added: REPLACEME
+-->
+
+Creates a new Tracker object.
```

Comment:
There's likely no reason to expose the constructor directly if we also have `createCallTracker()`.